mkdocs/tests/utils_tests.py

```diff
@@ -67,3 +68,34 @@ def test_create_media_urls(self):
         for path, expected_result in expected_results.items():
             urls = utils.create_media_urls(site_navigation, [path])
             self.assertEqual(urls[0], expected_result)
+
```

Comment:
Rather than try/catching in the unittest I'd prefer:

    @unittest.skipIf(sys.version_info[:2] == (2, 6))